nidirect_common/nidirect_common.links.menu.yml

```diff
@@ -20,7 +20,7 @@ nidirect.main.help:
   weight: 7
 nidirect.main.feedback:
   title: 'Feedback'
-  url: internal:/forms/feedback?s=/front
+  url: internal:/form/site-feedback
```

Comment:
Worth checking to see if they still want the `?s=/front` parameter in case it's relevant for GA tracking of some sort

Reply:
Good thought, I checked with Neil and he did want the parameter preserved.
Have now used the canonical path in there, please re-review